packages/admin-panel/cypress/e2e/survey/surveyImport.spec.js

```diff
@@ -12,15 +13,40 @@ import {
   selectReportingPeriod,
   selectDataService,
   importSurvey,
+  searchBySurveyName,
+  closeImportSurveyForm,
 } from '../../support';
 
-describe('import survey file', () => {
-  before(() => {
-    loginAsSuperUser();
-    openImportSurveyForm();
+beforeEach(() => {
+  loginAsSuperUser();
+  openImportSurveyForm();
+});
+
+describe('import new survey file', () => {
+  // Check error message.
+  it('Name does not match the sheet Name ', () => {
+    enterSurveyName('Test');
+    cy.uploadFile('surveys/Test Survey_1.xlsx');
+    importSurvey();
+    // assertion for error message.
+    cy.get('form').should('includes.text', 'Import failed');
+    cy.get('form').contains('Dismiss').click();
+    closeImportSurveyForm();
+  });
+
+  it('import a new survey by filling the mandatory fields', () => {
+    enterSurveyName('Test Survey_1');
+    cy.uploadFile('surveys/Test Survey_1.xlsx');
+    importSurvey();
+    // assertion for confirmation message.
+    cy.get('form').should('contain.text', 'Your import has been successfully processed');
+    cy.get('form').contains('Done').click();
+    searchBySurveyName('Test Survey_1');
+    // assertion for survey search.
+    cy.FirstRowElementOfTable().eq(1).should('have.text', 'Test Survey_1');
   });
 
-  it('import a single survey to a single country', () => {
+  it('import a new survey by filling all the fields', () => {
     enterSurveyName('Test Survey_1');
     enterCountryName('Demo Land');
     enterPermissionGroup('Admin');
```

Comment:
I believe the tests would benefit from a more generic util, eg

```js
fillImportSurveyForm({
  name: 'Test Survey_1',
  countryName: 'Demo Land'
});
```

And since we only display one form dialog at a time, this could actually be generalised so that it can be used in any admin panel test:

```js
fillDialogForm({
  Survey Name: 'Test Survey_1',
  Countries: ['Demo Land']
});
```

In which case we may decide its worth making this a command, since it will be present in plenty of admin panel tests (`cy.fillDialogForm`)